Libraries/oneVPL/dpcpp-blur/README.md

```diff
@@ -32,8 +33,11 @@ If the oneAPI DPC++ Compiler is not found, the blur operation will be disabled.
 | Target device     | CPU
 | Input format      | I420
 | Output format     | BGRA raw video elementary stream
-| Output resolution | same as input
+| Output resolution | 256x192
 
+## Linux GPU optimizations
+
```

Comment:
Can you explain what this does a little.  it's very out of context.  If you intend for the user to enter this command, it should be in the same code format as the commands for running the sample.

Reply:
Thanks for finding this -- this section was not for this version of the sample.  Removed.  For general GPU setup the user can follow the oneAPI basekit setup instructions.